Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -198,6 +198,7 @@ private void UpdatePosition(Vector3 input)
                     var groundNormal = GroundNormal();
                     var direction = Vector3.ProjectOnPlane(input, groundNormal);
                     body.AddForce(direction * strafeForce, ForceMode.VelocityChange);
+                    body.AddForce(direction * strafeForce, ForceMode.Impulse);
```

Comment:
also excuse me what? why is there an extra impulse force to the regular walking?

Reply:
Adressing feedback regarding acceleration when starting to move being too low. This change helps movement feel more "snappier".

I forgot to declare it in the PR description, will do it now